libs/styles/styles.css

```diff
@@ -152,6 +152,11 @@ img {
   height: auto;
 }
 
+/* Tooltip */
+em:has(.icon-tooltip, .icon-info) {
```

Comment:
@Blainegunn This (has() selector) is going to be a problem for older Safari. We currently support back to Safari 14

https://milo.adobe.com/wiki/test-automation/test-automation-framework#browser--platform-support-matrix

Reply:
i'm on it